src/connection.js

```diff
@@ -535,7 +535,7 @@ export class Connection {
   }
 
   /**
-   * Fetch the current total currency supply of the cluster
+   * Fetch the current total currency supply, in lamports, of the cluster
```

Comment:
Reads a little easier:
Fetch the current total currency supply of the cluster, in lamports

Reply:
Fixed. I promise not to tell where I copied that from... 😂 

https://github.com/sunnygleason/solana-web3.js/blame/patch-1/src/connection.js#L43

Reply:
Gimme a break, I'm from 🇨🇦

Reply:
That one reads okay... different sentence construction :)